README.md

```diff
@@ -148,6 +147,72 @@ token: xxxxxxxxxxxx # token of the telegram bot
 bot_tag: "@bot" # tag of the telegram bot
 ```
 
+### Settings override
+
+The settings may also be set through environment variables.  
+All the env vars with the same name (case insensitive) will override the ones in the settings file.
+To update the **meme** settings, prefix the env var name with **MEME\_**. The same is true for the **test** settings, that have to be prefixed with **TEST\_** and the **debug** settings, to be prefixed with **DEBUG\_**.
+
+```env
+# Environment values
+TOKEN=xxxxxx        # will override the *token* value found in settings.yaml
+MEME_N_VOTES=4      # will override the *meme.n_votes* value found in settings.yaml
+DEBUG_LOCAL_LOG=4   # will override the *debug.local_log* value found in settings.yaml
+```
+
+The complete order of precedence is for the application of configuration settings is
+
+```
+env var > settings.yaml > settings.yaml.default
+```
+
+Since every setting has a default value specified in _settings.yaml.default_ except for **token**, this is the only necessary settings to add when setting up the bot for the first time.
```

Comment:
Grammarly check suggestions
```suggestion
Since every setting has a default value specified in _settings.yaml.default_ except for **token**, this is the only necessary setting to add when setting up the bot for the first time.
```